### DISCUSSION TITLE
Page stuck when I close dialog or sheet... weird problem..

### DISCUSSION THREAD

**Discussion by abdulazizkhatamov** [shadcn-ui/ui#6116] [Q&A] [answered]
If I click edit button to open row data with shadcdn dialog or sheet for updating, it is opening well but once I close... my whole page/site is stuck.. can't doing anything...

important lines to check codes
Edit button: https://github.com/abdulazizkhatamov/exclusive-ecommerce/blob/6068c05a0182ab1dd9d13e9e56777cb299e4e307/admin/src/components/custom/table/categories/data-table-row-actions.tsx#L45
Dialog: https://github.com/abdulazizkhatamov/exclusive-ecommerce/blob/6068c05a0182ab1dd9d13e9e56777cb299e4e307/admin/src/elements/layouts/CategoriesLayout.tsx#L31

I used both dialog and sheet, almost tried all way, used sheet and dialog as react portal and more... but I still could not find why this is happening... 

Before clicking edit button:
![image](https://github.com/user-attachments/assets/0aa6f236-7122-430f-9667-6f3887342bcd)
After clicking edit button:
![image](https://github.com/user-attachments/assets/ad56b8af-fe96-494a-a02e-b4ada13456ea)
After closing edit button and nothing is working, whole page/site is stuck:
![image](https://github.com/user-attachments/assets/a0a79099-dcc9-4532-8da6-0d82536edd45)

**Comment by kr-arunp** [comment]
same issue

**Reply by abdulazizkhatamov** [reply, marked as answer]
Try to reinstall component

**Comment by clemkraw** [comment]
same issue too

**Reply by abdulazizkhatamov** [reply]
https://github.com/shadcn-ui/ui/discussions/6116?notification_referrer_id=NT_kwDOCtAZZrUxMzkxNTA1ODIxNzoxODE0MTAxNTA#discussioncomment-11781757

**Comment by jagvaandanzan** [comment]
for me `@radix-ui/react-dismissable-layer` this package was missing. After I installed it the bug has gone.

**Reply by roshan-gh** [reply]
It didn't fix for me with this package

**Reply by jagvaandanzan** [reply]
I think the issue was caused by the package versions. I tried reinstalling all the shadCN components and radix deps, and so far, there are no issues.

**Comment by adeludedperson** [comment]
as said in above comments, re-installing components worked fine for me.

**Reply by Kha-10** [reply]
re-installing components still doesn't work for me

**Comment by techcoder2007** [comment]
same issue too

**Comment by ibrahim-wael-ibrahim** [comment]
body {
    @apply bg-background text-foreground;
    pointer-events: auto !important;    <- this line 
  }
  
  can add this line to index.css

**Reply by eliuddyn** [reply]
This worked fine for me. 🔥🔥

**Reply by andrian-mampu** [reply]
yeah, this worked for me too..

**Reply by phalla-doll** [reply]
Try to re-install component didn't work for me. This worked by adding it to `globals.css`

**Reply by mrWealthid** [reply]
This worked for me... thanks a bunch

**Reply by mukhammadxuja** [reply]
This worked, thank you!

**Reply by noyonalways** [reply]
Yes, it is working for me.

**Reply by ravanedaa** [reply]
Worked For me! Thansk!

**Reply by mozi47** [reply]
Thanks, this works.

**Comment by aniqjaved01** [comment]
If you are still facing issue then you can refer to this: https://github.com/shadcn-ui/ui/issues/2525

**Reply by connor-ashton-dev** [reply]
This worked for me

**Reply by HatemSaleh** [reply]
this worked for me <DropdownMenu modal={false}> 
THANKS!!

**Comment by tomabadie** [comment]
Seems the origin of the issue has been found there : https://github.com/shadcn-ui/ui/discussions/6908#discussioncomment-14495997

It is caused by libraries using different versions of `@radix-ui/react-dissmissable-layer`.
Updating radix popover/alert/alert-dialog to the latest version currently works for me because they are using the same version : `1.1.11`.

If you can't update dependencies or the issue persists, I mentioned a workaround below the linked comment. It avoids `modal={false}` or any CSS hacks.
